website/pages/docs/k8s/installation/index.mdx

```diff
@@ -60,44 +60,31 @@ need to install Tiller by following the
 
 ### Installing Consul
 
-Determine the latest version of the Consul Helm chart
-by visiting [https://github.com/hashicorp/consul-helm/releases](https://github.com/hashicorp/consul-helm/releases).
+Add the HashiCorp Helm Repository:
 
-Clone the chart at that version. For example, if the latest version is
-`v0.8.1`, you would run:
-
-```bash
-$ git clone --single-branch --branch v0.8.1 https://github.com/hashicorp/consul-helm.git
-Cloning into 'consul-helm'...
-...
-You are in 'detached HEAD' state...
+```shell
+$ helm repo add hashicorp https://helm.releases.hashicorp.com
+"hashicorp" has been added to your repositories
 ```
 
-Ensure you've checked out the correct version with `helm inspect chart`:
+Ensure you have access to the consul chart:
 
-```bash
-$ helm inspect chart ./consul-helm
-apiVersion: v1
-description: Install and configure Consul on Kubernetes.
-home: https://www.consul.io
-name: consul
-sources:
-- https://github.com/hashicorp/consul
-- https://github.com/hashicorp/consul-helm
-- https://github.com/hashicorp/consul-k8s
-version: 0.8.1
+```shell
+$ helm search repo hashicorp/consul
+NAME            	CHART VERSION	APP VERSION	DESCRIPTION
+hashicorp/consul	0.20.1       	1.7.2      	Official HashiCorp Consul Chart
 ```
 
 Now you're ready to install Consul! To install Consul with the default
 configuration using Helm 3 run:
 
 ```shell
-$ helm install hashicorp ./consul-helm
-NAME: hashicorp
+$ helm install consul hashicorp/consul --set global.name=consul
```

Comment:
Previously we'd avoided setting example installation names to `consul` because that could result in `consul-consul-` prefixes on some resources.

Reply:
Though I see you've addressed that below 😂